src/uwtools/tests/test_schemas.py

```diff
@@ -1399,7 +1449,7 @@ def test_schema_upp_run_dir(upp_prop):
     assert "88 is not of type 'string'" in errors(88)
 
 
-# ungrib
+# wavewatch III
```

Comment:
```suggestion
# ww3
```
I think these headings are meant to track the `.jsonschema` filenames, though they are not currently all correct (e.g. underscores instead of hyphens). I put harmonizing those on my todo list, but `ww3` would be correct in any case.